tests/test_data_adapter.py

```diff
@@ -78,9 +78,9 @@ def test_rasterdataset(rioda, tmpdir):
     geom = rioda.raster.box
     da1 = data_catalog.get_rasterdataset("test.tif", geom=geom)
     assert np.all(da1 == rioda_utm)
-    with pytest.raises(FileNotFoundError, match="No such file"):
+    with pytest.raises(NoDataException):
```

Comment:
See my comment on unmerging the error messages for FIleNotFound and NoDataException